crates/turbopack/src/resolve_options_context.rs

```diff
@@ -17,7 +18,9 @@ pub struct ResolveOptionsContext {
     pub enable_typescript: bool,
     pub enable_react: bool,
     pub enable_node_native_modules: bool,
-    pub enable_node_modules: bool,
+    /// Enable resolving of the node_modules folder when withing the provided
+    /// directory
+    pub enable_node_modules: Option<FileSystemPathVc>,
```

Comment:
```suggestion
    pub node_modules_directory: Option<FileSystemPathVc>,
```
This should no longer read `enable_smth`, but be called `node_modules_directory` directly.